pyproject.toml

```diff
@@ -43,7 +44,9 @@ keywords = [
 readme = "README.md"
 
 [tool.poetry.dependencies]
-setuptools_scm = { version = "^8.0", python = ">=3.8,<4" }
+hatch = { version = "^1.9.1", python = ">=3.9,<4" }
+hatchling = { version = "^1.21.0", python = ">=3.8,<4" }
+hatch-semver = { version = "^1.0.1",  python = ">=3.9,<4" }
```

Comment:
Excellent work! We need to drop Python 3.7 support before merging this, though. #37